Podfile

```diff
@@ -27,9 +27,9 @@ def aztec
 end
 
 def tracks
-  pod 'Automattic-Tracks-iOS', '~> 2.2'
+#  pod 'Automattic-Tracks-iOS', '~> 2.2'
   # pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :branch => 'trunk'
-  # pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :commit => ''
+   pod 'Automattic-Tracks-iOS', :git => 'https://github.com/Automattic/Automattic-Tracks-iOS.git', :commit => '1d3bb8030b3aed54c2b24c18fb95f3945ac63f9f'
```

Comment:
Looks like there have been a few updates [here](https://github.com/Automattic/Automattic-Tracks-iOS/pull/261/commits). Should we point it to the latest commit, `d841ba6b6a048218a8d8507b8d71c0c1349036ae`?

Reply:
Since the PR has been merged, I've point the library to `trunk` now. I'll update it to a version number once the update is released.

Reply:
Since I cannot trigger CI to build the updated version of the library on Cocoapods, I'll merge this PR for now to make it to the code freeze (pointing the library to `trunk`). Hopefully, a release manager would help with updating the library version cc @spencertransier 🙇